src/standalone/Vegetation/photosynthesis.jl

```diff
@@ -1,4 +1,33 @@
-export FarquharParameters, FarquharModel, C3, C4
+export SIFParameters, FarquharParameters, FarquharModel, C3, C4
+
+abstract type AbstractPhotosynthesisModel{FT} <: AbstractCanopyComponent{FT} end
+
+"""
+    SIFParameters{FT<:AbstractFloat}
+
+The required parameters for the SIF parameterisation [give citation].
+$(DocStringExtensions.FIELDS)
+"""
+@kwdef struct SIFParameters{FT <: AbstractFloat}
+    ""
+    kf::FT = FT(0.05)
+    ""
+    kd_p1::FT = FT(0.03)
```

Comment:
Parameter used to compute the rate coefficient for heat loss in dark-adapted conditions, Tol et al. 2014